config.json

```diff
@@ -33,6 +33,17 @@
       "unlocked_by": "two-fer",
       "uuid": "2c8afeed-480e-41f3-ad58-590fa8f88029"
     },
+    {
+      "core": false,
+      "difficulty": 2,
+      "slug": "armstrong-numbers",
+      "topics": [
+        "integers",
+        "mathematics"
+      ],
+      "unlocked_by": null,
```

Comment:
Maybe this should be unlocked by some other exercise? Or do you think it should be always accessible? I think either is fine :)